.hooks/verify-zarf-schema.sh

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env sh
 go run main.go internal config-schema > zarf.schema.json
+go run main.go internal override-schema > zarf-override.schema.json
```

Comment:
Looks like this is not the only place this is being done. See https://github.com/defenseunicorns/zarf/blob/master/.github/workflows/zarf-schema-check.yml#L39

We should either add this new stuff to there too, or make the other place use this shell script, so that we are more DRY

Reply:
Yeah this probably fits in the makefile and then called by the action/hook.

Reply:
Changed the way this works, it now checks all docs (including CLI docs) and is integrated into make